examples/neg/namer/issue950/ambiguous_interfaces_overload.check

```diff
@@ -0,0 +1,9 @@
+[info] /home/jakub/Code/Effekt/effekt/examples/neg/namer/issue950/ambiguous_interfaces_overload.effekt:5:5: There is an equally named effect operation sayHello of interface Greet. Use syntax `do sayHello()` to call it.
+    sayHello() 
+    ^^^^^^^^
+[info] /home/jakub/Code/Effekt/effekt/examples/neg/namer/issue950/ambiguous_interfaces_overload.effekt:5:5: There is an equally named effect operation sayHello of interface Welcome. Use syntax `do sayHello()` to call it.
+    sayHello() 
+    ^^^^^^^^
+[error] /home/jakub/Code/Effekt/effekt/examples/neg/namer/issue950/ambiguous_interfaces_overload.effekt:5:5: Cannot find a function named `sayHello`.
+    sayHello() 
+    ^^^^^^^^
```

Comment:
In the checkfiles, the path of the Effekt file should only be the one from `examples` forward, like in, let's say `examples/neg/namer/issue950/cannot_find_function.check`